src/sentry/models/featureadoption.py

```diff
@@ -102,16 +103,20 @@ def get_all_cache(self, organization_id):
         return {int(x) for x in set.union(*[p.value for p in result])}
 
     def bulk_set_cache(self, organization_id, *args):
+        if not args:
+            return
```

Comment:
Damn. I ended up accidentally committing to the old ` feature-adoption-receivers` branch and cherry picked the commit, so I missed these. Thanks for catching.